bbq/vm/linker.go

```diff
@@ -123,16 +123,23 @@ func LinkGlobals(
 			continue
 		}
 
-		funcStaticType := getTypeFromExecutable[interpreter.FunctionStaticType](executable, function.TypeIndex)
-
-		value := FunctionValue{
-			Function:   function,
-			Executable: executable,
-			Type:       funcStaticType,
+		var value interpreter.FunctionValue
+
+		if function.IsNative() {
+			// Look-up using the unqualified name, in the common-builtin functions.
+			value = IndexedCommonBuiltinTypeBoundFunctions[function.Name]
```

Comment:
When linking, if the function/method is natively provided, then look it up from the pre-known list.